render.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
can we get a comment on what this shell script is supposed do? I assume it just repalces the version in the dockerfile.template.

Reply:
Except it can do that for many vesions? Why would we want many versions?

Reply:
Seems you mentioned multiple  on the last pull request but I'm still not getting it. Add to readme?